LayoutTests/compositing/shared-backing/bug276216.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ OverlappingBackingStoreProvidersEnabled=true ] -->
```

Comment:
This hopefully isn't really necessary (preferences that are 'testable' or higher should be enabled by default).